doc/man/man3/seccomp_action_valid.3

```diff
@@ -22,7 +22,9 @@ The
 function validates that the given seccomp filter
 .I action
 is known to libseccomp and, in the case of newer actions, supported by the
-kernel that's currently running.
+kernel that's currently running. This function is affected by the
+.BR SCMP_GLBATR_CTL_KCHECKACTS
+attribute which will disable the kernel support check for newer actions.
```

Comment:
This goes back to our email discussion about checking/setting a "API level" instead of checking/setting specific "things".  I'm not saying "no" to this approach yet, but I would like to see an "API level" approach before making a call on this.